vivisect/symboliks/common.py

```diff
@@ -582,7 +585,11 @@ def getWidth(self):
         return self.kids[1].solve()
 
     def _reduce(self, emu):
-        return Mem(self.kids[0].reduce(), self.kids[1].reduce())
+        addr = self.kids[0]._reduce(emu=emu)
```

Comment:
this just feels wrong.  are there any explanations?  why is this necessary?  _reduce shouldn't return None, right?  or am i getting my paradigms mixed up?

Reply:
Paradigm mixup I think. As far as I can tell, the base implementation of _reduce (and _op_reduce) have always returned None.